src/components/home/approach.tsx

```diff
@@ -189,25 +222,55 @@ const StyledIcon = styled(ArrowUpRight)`
 
   transition: 0.3s ease;
 
-  &:hover {
-    transform: rotate(45deg);
-  }
 
   @media all and (max-width: ${breakpoints.md}px) {
-    top: 38%; /* adjust top for medium screens */
-    right: 6%; /* adjust right for medium screens */
+    top: 38%;
+    right: 6%;
   }
 
   @media all and (max-width: ${breakpoints.sm}px) {
-    top: 29%; /* adjust top for small screens */
-    right: 6%; /* adjust right for small screens */
+    top: 30%;
+    right: 4%;
+`;
+
+const StyledPillButton = styled(motion.div)`
```

Comment:
<img width="489" alt="Screenshot 2024-04-04 at 17 01 32" src="https://github.com/web-space-dev/website-frontend/assets/29145846/63f55257-074c-4918-b3ec-bbffac15280d">

On medium screens e.g. 520px, can we have a bit more spacing between the text and the icon